### ISSUE TITLE
Deployment helper functions fail

### ISSUE THREAD

**Issue by georgeroman** [balancer-labs/balancer-v2-monorepo#802]
I'm trying to use the `getBalancerContract` helper function for retrieving an instance of the V2 Vault contract, but it seems to be failing when requiring the JSON file containing the address. I didn't test the other functions but I assume they all fail with the same error.

Here is a sample code snippet to replicate:
```javascript
const { getBalancerContract } = require("@balancer-labs/v2-deployments");

async function main() {
  const vault = await getBalancerContract("20210418-vault", "Vault", "mainnet");
}

main()
  .then(() => process.exit(0))
  .catch((error) => {
    console.error(error.message);
    process.exit(1);
  });
```

And here is the error I'm getting:
```bash
Cannot find module '@balancer-labs/v2-deployments/tasks/20210418-vault/output/mainnet.json'
Require stack:
- /home/george/Development/misc/test-deployments/node_modules/@balancer-labs/v2-deployments/dist/index.js
- /home/george/Development/misc/test-deployments/index.js
```

**Comment by nventuro** [comment]
Hello @georgeroman, thank you very much for reporting this!

Indeed there was a slight issue in the path of the artifact files, it's now been fixed on the latest published version, v2.1.2. Would you mind giving that a try and reporting back?

Also, note that in order to use `getBalancerContractAt` you'll need to also install hardhat and the hardhat-ethers plugins (which are listed as `peerDependencies`, as that function returns an Ethers contract object.

**Comment by nventuro** [comment]
Thanks @georgeroman  - indeed there was a second unrelated issue. Both have been addressed in v2.1.3, which I've tested locally and seems to work fine. Let me know if you run into any further issues!

**Comment by georgeroman** [comment]
Thanks again for the quick fix, can confirm it's working properly now.
